src/h5cpp/node/dataset.cpp

```diff
@@ -189,10 +189,10 @@ void Dataset::refresh() const
 
 #if H5_VERSION_GE(1,10,2)
 
-long long unsigned int Dataset::chunk_storage_size(
-			 std::vector<long long unsigned int> offset) const
+unsigned long long Dataset::chunk_storage_size(
```

Comment:
what about using uint64_t instead of unsigned long long

Reply:
In the API of libhdf5 the offset is defined with `hsize_t`. I will try to switch to it